### PR TITLE
Feature/2017 12 03 xp days benelux 2017

### PR DESCRIPTION
**Pull request by Deryaa**
TODO: change publication date

### CODE REVIEW

File: _posts/2017-12-03-XPDays-Benelux-2017.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+### Motivate Your Team with Gamification - Jean-Jacques Courtens
+
+<span class="image left"><img class="p-image" alt="Jean-Jacques Courtens" src="/img/xpdays-benelux-2017/JJ-Courtens.png"></span>
+
+[Jean-Jacques Courtens](https://twitter.com/jjcourtens){:target="_blank"} is the Founder and a Managing Partner at Adsdaq where he introduced gamification as a means of motivating the teams to deliver working products consistently.
+
+This was an Open Space session where Jean-Jacques discussed the way he implemented gamification in his company. At Adsdaq, they work with a reward system where points can be earned by completing certain tasks.
+They assign points to three sections: Timesheets, demos and sprint objectives.
+* Timesheets: Filling in the timesheet in time is worth **1 point**
+* Demos: Doing 1 demo is worth **2 points**
```
Comment (yannickdeturck) [inline _posts/2017-12-03-XPDays-Benelux-2017.md]: "Doing a demo" sounds better?

File: _posts/2017-12-03-XPDays-Benelux-2017.md [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+It's also important to note that the team is still being rewarded when the sprint fails and not being reprimanded for it. The reward is just a lot bigger when the sprints succeed.
+
+### An Integral View on Agile - Frederik Vannieuwenhuyse & Johannes Schartau
+
+<span class="image left"><img class="p-image" alt="Frederik Vannieuwenhuyse" src="/img/xpdays-benelux-2017/Frederik-Vannieuwenhuyse.png"></span>
+<span class="image left"><img class="p-image" alt="Johannes Schartau" src="/img/xpdays-benelux-2017/Johannes-Schartau.png"></span>
+[Frederik Vannieuwenhuyse](https://twitter.com/vfrederik) is a multidisciplinary generalising specialist and is continuously on a journey of discovery and learning how to grow effective, resilient and agile organisations.<br><br>
+[Johannes Schartau](https://twitter.com/integralagile) is an Agile Coach, consultant and professional Zombie Scrum fighter from Hamburg, Germany. He is passionate about creating environments for real collaboration and joyful creativity.
+
+The first part of this exercise consisted of reading 6 case studies and writing down on post-it notes how you would handle each situation, what questions you would ask.
```
Comment (yannickdeturck) [inline _posts/2017-12-03-XPDays-Benelux-2017.md]: 6 -> six

File: _posts/2017-12-03-XPDays-Benelux-2017.md [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+<div class="row">
+  <div class="6u -3u 8u(small) -2u(small) 12u(xsmall)">
+    <img class="image fit" alt="The 8 Misunderstood Stances of a Scrum Master" src="/img/xpdays-benelux-2017/Integral-Theory2.png">
+  </div>
+</div>
+
+After explaining Integral Theory, we picked up our post-it notes and organized them into the quadrants that were put up across the room.
+When the post-it notes were assigned to their respective quadrant, Frederik and Johannes asked us to move towards the quadrant in the room that we felt we had the most experience in or we felt we were good at.
+Next, we had to discuss in our 'quadrant groups' if the post-it notes that were put up in the quadrant were a good match for it or if they belonged in another quadrant.
+The final exercise consisted of moving with your group to the next quadrants and listening to 1 person that stayed behind from the initial group explain the conclusions they reached during the initial discussion.
```
Comment (yannickdeturck) [inline _posts/2017-12-03-XPDays-Benelux-2017.md]: ... one person, that stayed behind from the initial group, explain ...

File: _posts/2017-12-03-XPDays-Benelux-2017.md [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+The point of the interview was asking questions to guide the interviewee in telling their story. After these interviews, the person interviewing had to share what they had learned with the rest of the table and the others had to take notes of the summary.
+The steps for executing this exercise are covered in this picture:
+<div class="row">
+  <div class="4u -4u 6u(small) -3u(small) 8u(xsmall) -2u(xsmall)">
+    <img class="image fit" alt="The 8 Misunderstood Stances of a Scrum Master" src="/img/xpdays-benelux-2017/Art-of-Hosting2.png">
+  </div>
+</div>
+
+After this exercise, it was already clear that asking appropriate and considerate questions is a key element of hosting a good and constructive conversation.
+This kind of conversation is built upon 4 key elements:
```
Comment (yannickdeturck) [inline _posts/2017-12-03-XPDays-Benelux-2017.md]: 4 -> four

File: _posts/2017-12-03-XPDays-Benelux-2017.md [hunk 5]
```diff
@@ -0,0 +1,208 @@
...
+* **Admin:** The Scrum Master is the workflow master. Need to add a board in Jira? Ask the Scrum Master. Need to start a sprint in Jira? Ask the Scrum Master. Now, hold up right there! The Scrum Master *can* of course perform these tasks but yup, so can the other team members.
+* **Secretary:** The Scrum Master plans all the work in the team members' agendas. Keep up to date with everybody's holidays, sick days and toilet breaks. (Can you sense the sarcasm?)
+* **Chairman:** While this is often the case, the Daily Scrum isn't a meeting where the team members report back to the Scrum Master. It's a meeting by and for the development team. In absence of the Scrum Master, the Daily Scrum must still happen.
+* **Team Boss:** The Scrum Master is the boss of the team. He/she decides who is in and who is out. A sick day? You're fired! Buy the Scrum Master a chocolate cake? Here, have a raise! Oh, what a world it would be!
+* **Coffee Clerk:** Please do get your team members some coffee once in a while. But no, to everyone's surprise, fetching coffee day in day out is not part of the Scrum Master's job description.
+
+So, what are the 8 preferred stances of a Scrum Master then?
```
Comment (yannickdeturck) [inline _posts/2017-12-03-XPDays-Benelux-2017.md]: 8 -> eight

File: _posts/2017-12-03-XPDays-Benelux-2017.md [hunk 6]
```diff
@@ -0,0 +1,208 @@
...
+Glad you asked!
+
+The 8 preferred stances are:
```
Comment (yannickdeturck) [inline _posts/2017-12-03-XPDays-Benelux-2017.md]: 8 -> eight

File: _posts/2017-12-03-XPDays-Benelux-2017.md [hunk 7]
```diff
@@ -0,0 +1,208 @@
...
+
+<div class="row">
+  <div class="4u -4u 6u(small) -3u(small) 8u(xsmall) -2u(xsmall)">
+    <img class="image fit" alt="The 8 Preferred Stances of a Scrum Master" src="/img/xpdays-benelux-2017/8-Preferred-Stances.png">
+  </div>
+</div>
+
+* **Teacher:** "The art of teaching is the art of assisting discovery." - Mark Van Doren. There is much the Scrum Master can teach the Scrum Team. The Scrum Master must ensure that Scrum is understood and implemented properly by the entire team.
+He or she makes sure the team stays on track of the Agile practices and principles.
+* **Impediment Remover:** An Impediment is a problem that goes beyond the self-organization of the Development Team. Make sure the team understands and uses their own ability to solve problems and be self-organized. It might also create an opportunity for the team to come up with creative ideas to solve the impediments themselves.
```
Comment (yannickdeturck) [inline _posts/2017-12-03-XPDays-Benelux-2017.md]: An **i**mpediment is ...(lowercase 'i' here?)

File: _data/authors.yml [hunk 8]
```diff
@@ -442,4 +431,15 @@ bas_moorkens:
...
+  first_name: Derya
+  last_name: Duru
+  permalink: /author/derya-duru/
+  title: Scrum Master
+  email: derya.duru@ordina.be
+  linkedin: derya-duru-52348a3a
+  avatar: derya-duru.jpg
+  twitter: deryaduru_
+  github: Deryaa
+  bio: Derya is a Scrum Master at Ordina Belgium always trying to better herself in Agile frameworks. She enjoys being challenged and as the Competence Leader Agile, she tries to improve her own skills continuously while hoping to share her knowledge and help others with their understanding of Agile frameworks.
```
Comment (bart-blommaerts) [inline _data/authors.yml]: continuously, while

File: _posts/2017-12-18-XPDays-Benelux-2017.md [hunk 9]
```diff
@@ -0,0 +1,208 @@
...
+layout: post
+authors: [derya_duru]
+title: 'XP Days Benelux 2017'
+image: /img/xpdays-benelux-2017/XPDays-Benelux-2017.png
+tags: [XP Days Benelux, Agile, Scrum, Scrum Master, Gamification, Integral Theory, Art of Hosting, Facilitation, Coaching]
+category: Conference
+comments: true
+---
+
+> [XP Days Benelux](http://www.xpdaysbenelux.nl){:target="_blank"} is a two day conference on agile software development for and by practitioners.
```
Comment (bart-blommaerts) [inline _posts/2017-12-18-XPDays-Benelux-2017.md]: for and by **agile** practitioners?

File: _posts/2017-12-18-XPDays-Benelux-2017.md [hunk 10]
```diff
@@ -0,0 +1,208 @@
...
+</div>
+
+****
+
+### The 8 Stances of a Scrum Master - Barry Overeem
+
+<span class="image left"><img class="p-image" alt="Barry Overeem" src="/img/xpdays-benelux-2017/Barry-Overeem.jpg"></span>
+[Barry Overeem](https://twitter.com/barryovereem){:target="_blank"} is a freelance Scrum Master and Professional Scrum Trainer at Scrum.org. He’s an active member of the Scrum community and shares his insights and knowledge by speaking at conferences, facilitating workshops and writing blog posts.
+
+It is so common for a Scrum Master to get assigned tasks that aren't actually tasks a Scrum Master should be performing. The job description of a Scrum Master comes with whole other tasks than a lot of people assume, sometimes even the Scrum Master him or herself.
```
Comment (bart-blommaerts) [inline _posts/2017-12-18-XPDays-Benelux-2017.md]: very common?
or just "common"?

"so common" sounds strange

File: _posts/2017-12-18-XPDays-Benelux-2017.md [hunk 11]
```diff
@@ -0,0 +1,208 @@
...
+authors: [derya_duru]
+title: 'XP Days Benelux 2017'
+image: /img/xpdays-benelux-2017/XPDays-Benelux-2017.png
+tags: [XP Days Benelux, Agile, Scrum, Scrum Master, Gamification, Integral Theory, Art of Hosting, Facilitation, Coaching]
+category: Conference
+comments: true
+---
+
+> [XP Days Benelux](http://www.xpdaysbenelux.nl){:target="_blank"} is a two day conference on agile software development for and by practitioners.
+In this post, I will go over some talks and sessions I attended and participated in.
```
Comment (HPacquee) [inline _posts/2017-12-18-XPDays-Benelux-2017.md]: In this post, I will take you along to some talks I attended and some sessions I participated in.

File: _posts/2017-12-18-XPDays-Benelux-2017.md (hunk 10 above)
Comment (bart-blommaerts) [inline _posts/2017-12-18-XPDays-Benelux-2017.md]: 1 sentence per line would be awesome :)

Comment (bart-blommaerts) [reply _posts/2017-12-18-XPDays-Benelux-2017.md]: I'd rephrase to something like:

It is very common for a Scrum Master to get tasks assigned, that aren't actually tasks a Scrum Master should be performing in the first place. 
The job description of a Scrum Master comes with a lot of other tasks than people, sometimes including the Scrum Master, assume.

File: _posts/2017-12-18-XPDays-Benelux-2017.md [hunk 12]
```diff
@@ -0,0 +1,208 @@
...
+
+The session starts with an overview of the 8 misunderstood stances of a Scrum Master:
+<div class="row">
+  <div class="4u -4u 6u(small) -3u(small) 8u(xsmall) -2u(xsmall)">
+    <img class="image fit" alt="The 8 Misunderstood Stances of a Scrum Master" src="/img/xpdays-benelux-2017/8-Misunderstood-Stances.png">
+  </div>
+</div>
+
+* **Scrum Police:** Scrum isn't a hard set of rules to be followed. There's absolutely nothing wrong with a bit of flexibility and empathy based on the team's situation.
+Time boxing for example is very important but keep an open mind when the team is having a very valuable conversation when the time box has ended.
```
Comment (bart-blommaerts) [inline _posts/2017-12-18-XPDays-Benelux-2017.md]: conversation, after the estimated time has

File: _posts/2017-12-18-XPDays-Benelux-2017.md (hunk 10 above)
Comment (HPacquee) [inline _posts/2017-12-18-XPDays-Benelux-2017.md]: People don't expect the actual job content of a Scrum Master to contain such a wide array of tasks. Often times these tasks are even missed or forgotten by Scrum Masters themselves.

File: _posts/2017-12-18-XPDays-Benelux-2017.md [hunk 13]
```diff
@@ -0,0 +1,208 @@
...
+Flexibility regarding time boxing is of course not meant to happen too often. So keep an eye out for recurring discussions to see if they need to be addressed separately.
```
Comment (bart-blommaerts) [inline _posts/2017-12-18-XPDays-Benelux-2017.md]: keep an eye on recurring

addressed individually?

File: _posts/2017-12-18-XPDays-Benelux-2017.md [hunk 14]
```diff
@@ -0,0 +1,208 @@
...
+* **Hero:** Managing and solving all problems and impediments like nobody's business! While an important task for the Scrum Master, make sure not to get too focused on being the team's impediment super hero.
+* **Scribe:** "Hey John, you're taking notes again for this meeting, right? Thanks!". Do you recognize this exchange? Congratulations, you're the team's personal scribe.
+This is especially to be avoided during retrospectives since this creates a false sense of ownership of the issues and actions towards the Scrum Master.
+* **Admin:** The Scrum Master is the workflow master. Need to add a board in Jira? Ask the Scrum Master. Need to start a sprint in Jira? Ask the Scrum Master. Now, hold up right there! The Scrum Master *can* of course perform these tasks but yup, so can the other team members.
+* **Secretary:** The Scrum Master plans all the work in the team members' agendas. Keep up-to-date with everybody's holidays, sick days and toilet breaks. (Can you sense the sarcasm?)
```
Comment (bart-blommaerts) [inline _posts/2017-12-18-XPDays-Benelux-2017.md]: _(Can you sense the sarcasm?)_

italic

File: _posts/2017-12-18-XPDays-Benelux-2017.md [hunk 15]
```diff
@@ -0,0 +1,208 @@
...
+* **Chairman:** While this is often the case, the Daily Scrum isn't a meeting where the team members report back to the Scrum Master. It's a meeting by and for the development team. In absence of the Scrum Master, the Daily Scrum must still happen.
```
Comment (bart-blommaerts) [inline _posts/2017-12-18-XPDays-Benelux-2017.md]: Fuck yeah!

(no comments)

File: _posts/2017-12-18-XPDays-Benelux-2017.md [hunk 16]
```diff
@@ -0,0 +1,208 @@
...
+
+The eight preferred stances are:
+
+<div class="row">
+  <div class="4u -4u 6u(small) -3u(small) 8u(xsmall) -2u(xsmall)">
+    <img class="image fit" alt="The 8 Preferred Stances of a Scrum Master" src="/img/xpdays-benelux-2017/8-Preferred-Stances.png">
+  </div>
+</div>
+
+* **Teacher:** "The art of teaching is the art of assisting discovery." - Mark Van Doren. There is much the Scrum Master can teach the Scrum Team. The Scrum Master must ensure that Scrum is understood and implemented properly by the entire team.
```
Comment (bart-blommaerts) [inline _posts/2017-12-18-XPDays-Benelux-2017.md]: You can quote with "> " should you want.

File: _posts/2017-12-18-XPDays-Benelux-2017.md [hunk 17]
```diff
@@ -0,0 +1,208 @@
...
+* Third step: "What do you think will help you?" Guide the person towards a **change in language**. It's important they can see that a change in the situation is actually possible.
+* Fourth step: "When?" Create concrete **plans** for change. Don't postpone taking action. Plan it and make it happen.
+
+More information can be found on the Dutch website ['Praktisch op weg naar Teamresultaat'](http://www.praktischopweg.nl/){:target="_blank"}.
+
+### Motivate Your Team with Gamification - Jean-Jacques Courtens
+
+<span class="image left"><img class="p-image" alt="Jean-Jacques Courtens" src="/img/xpdays-benelux-2017/JJ-Courtens.png"></span>
+
+[Jean-Jacques Courtens](https://twitter.com/jjcourtens){:target="_blank"} is the Founder and a Managing Partner at Adsdaq where he introduced gamification as a means of motivating the teams to deliver working products consistently.
```
Comment (bart-blommaerts) [inline _posts/2017-12-18-XPDays-Benelux-2017.md]: founder and managing partner

File: _posts/2017-12-18-XPDays-Benelux-2017.md [hunk 18]
```diff
@@ -0,0 +1,208 @@
...
+
+### An Integral View on Agile - Frederik Vannieuwenhuyse & Johannes Schartau
+
+<span class="image left"><img class="p-image" alt="Frederik Vannieuwenhuyse" src="/img/xpdays-benelux-2017/Frederik-Vannieuwenhuyse.png"></span>
+<span class="image left"><img class="p-image" alt="Johannes Schartau" src="/img/xpdays-benelux-2017/Johannes-Schartau.png"></span>
+[Frederik Vannieuwenhuyse](https://twitter.com/vfrederik){:target="_blank"} is a multidisciplinary generalising specialist and is continuously on a journey of discovery and learning how to grow effective, resilient and agile organisations.<br><br>
+[Johannes Schartau](https://twitter.com/integralagile){:target="_blank"} is an Agile Coach, consultant and professional Zombie Scrum fighter from Hamburg, Germany. He is passionate about creating environments for real collaboration and joyful creativity.
+
+The first part of this exercise consisted of reading six case studies and writing down on post-it notes how you would handle each situation, what questions you would ask.
+These post-it notes will be used in a later exercise, after an explanation of the so-called Integral Theory:
```
Comment (bart-blommaerts) [inline _posts/2017-12-18-XPDays-Benelux-2017.md]: were used

(you also use past tense in "exercise consisted"

File: _posts/2017-12-18-XPDays-Benelux-2017.md [hunk 19]
```diff
@@ -0,0 +1,208 @@
...
+<div class="row">
+  <div class="6u -3u 8u(small) -2u(small) 12u(xsmall)">
+    <img class="image fit" alt="Integral Theory" src="/img/xpdays-benelux-2017/Integral-Theory1.png">
+  </div>
+</div>
+Integral Theory was created and shaped by Ken Wilber and consists of the idea that anything can be viewed as a thing by itself or as part of a larger group (**Individual VS Collective**).
```
Comment (bart-blommaerts) [inline _posts/2017-12-18-XPDays-Benelux-2017.md]: Link to Wilber?

File: _posts/2017-12-18-XPDays-Benelux-2017.md [hunk 20]
```diff
@@ -0,0 +1,208 @@
...
+The following image gives an overview of the questions that help organize knowledge into the quadrants:
+<div class="row">
+  <div class="6u -3u 8u(small) -2u(small) 12u(xsmall)">
+    <img class="image fit" alt="Integral Theory" src="/img/xpdays-benelux-2017/Integral-Theory2.png">
+  </div>
+</div>
+
+After explaining Integral Theory, we picked up our post-it notes and organized them into the quadrants that were put up across the room.
+When the post-it notes were assigned to their respective quadrant, Frederik and Johannes asked us to move towards the quadrant in the room that we felt we had the most experience in or we felt we were good at.
+Next, we had to discuss in our 'quadrant groups' if the post-it notes that were put up in the quadrant were a good match for it or if they belonged in another quadrant.
```
Comment (bart-blommaerts) [inline _posts/2017-12-18-XPDays-Benelux-2017.md]: whether the post-it notes ... or whether

(I think)

File: _posts/2017-12-18-XPDays-Benelux-2017.md [hunk 21]
```diff
@@ -0,0 +1,208 @@
...
+
+### The Art of Hosting Conversations that Matter - Johan Decoster & Jef Cumps
+
+<span class="image left"><img class="p-image" alt="Johan Decoster" src="/img/xpdays-benelux-2017/Johan-Decoster.png"></span>
+<span class="image left"><img class="p-image" alt="Jef Cumps" src="/img/xpdays-benelux-2017/Jef-Cumps.png"></span>
+[Johan Decoster](https://twitter.com/johan_decoster){:target="_blank"} is an Agile coach and trainer trying to make a difference in the lives of the people he works with by uncovering everyone’s unique potential and looking deeper into the essence behind any theory or concept.<br><br>
+[Jef Cumps](https://twitter.com/jcumps){:target="_blank"} is a very experienced coach and trainer supporting organisations in their transformation towards more agility and towards a more engaging, humane and effective way of looking at work.
+
+The session by Johan and Jef started with a small exercise. We paired up with our neighbor and asked each other the question, 'Why are you really at this conference, and what has it meant to you so far?'
```
Comment (bart-blommaerts) [inline _posts/2017-12-18-XPDays-Benelux-2017.md]: neighbour

File: _posts/2017-12-18-XPDays-Benelux-2017.md [hunk 22]
```diff
@@ -0,0 +1,208 @@
...
+The point of the interview was asking questions to guide the interviewee in telling their story. After these interviews, the person interviewing had to share what they had learned with the rest of the table and the others had to take notes of the summary.
```
Comment (bart-blommaerts) [inline _posts/2017-12-18-XPDays-Benelux-2017.md]: what he or she had learned with the rest of us.
We were then asked to take notes of the summary.

File: _posts/2017-12-18-XPDays-Benelux-2017.md [hunk 23]
```diff
@@ -0,0 +1,208 @@
...
+He or she makes sure the team stays on track of the Agile practices and principles.
+* **Impediment Remover:** An impediment is a problem that goes beyond the self-organization of the Development Team. Make sure the team understands and uses their own ability to solve problems and be self-organized. It might also create an opportunity for the team to come up with creative ideas to solve the impediments themselves.
+* **Facilitator:** Being a Scrum Master also means facilitating the team in transparency, inspection and adaptation. It is also through great facilitation that the Scrum Master succeeds in getting more value out of every event.
+* **Coach:** It's important to stay away from the solution and ask questions in order to facilitate discovery of solutions. Of course, the Scrum Master may offer new perspectives to help the team reach a solution.
+* **Servant Leader:** Remember that the Scrum Master serves others. Being able to read the room and facilitate conflict within the team is a very important responsibility of the Scrum Master. Make sure to lead by example and make others feel comfortable with failing.
```
Comment (HPacquee) [inline _posts/2017-12-18-XPDays-Benelux-2017.md]: resolve conflict (tenzij echt conflict faciliteren?)

File: _posts/2017-12-18-XPDays-Benelux-2017.md [hunk 24]
```diff
@@ -0,0 +1,208 @@
...
+The session by Johan and Jef started with a small exercise. We paired up with our neighbor and asked each other the question, 'Why are you really at this conference, and what has it meant to you so far?'
+The point of the interview was asking questions to guide the interviewee in telling their story. After these interviews, the person interviewing had to share what they had learned with the rest of the table and the others had to take notes of the summary.
+The steps for executing this exercise are covered in this picture:
+<div class="row">
+  <div class="4u -4u 6u(small) -3u(small) 8u(xsmall) -2u(xsmall)">
+    <img class="image fit" alt="The Art of Hosting" src="/img/xpdays-benelux-2017/Art-of-Hosting2.png">
+  </div>
+</div>
+
+After this exercise, it was already clear that asking appropriate and considerate questions is a key element of hosting a good and constructive conversation.
```
Comment (bart-blommaerts) [inline _posts/2017-12-18-XPDays-Benelux-2017.md]: it was clear that

File: _posts/2017-12-18-XPDays-Benelux-2017.md [hunk 25]
```diff
@@ -0,0 +1,208 @@
...
+  <div class="4u -4u 6u(small) -3u(small) 8u(xsmall) -2u(xsmall)">
+    <img class="image fit" alt="The Art of Hosting" src="/img/xpdays-benelux-2017/Art-of-Hosting3.png">
+  </div>
+</div>
+
+Make sure the conversation stays polite and has a good flow of dialogue, but also take note of the two other sections.
+It's very important to respect the differences in ideas between you and your conversation partner. It's okay to voice your opinion, but be considerate and respectful and make sure to not overpower the other person.
+Listen often and do not judge too quickly. The other person might not be good in voicing their opinion clearly so suspending judgement is always a good idea. Things may become clearer later on in the conversation.
+
+Right before the break, people were invited to write down questions they had where they could use the help of others to solve it.
```
Comment (bart-blommaerts) [inline _posts/2017-12-18-XPDays-Benelux-2017.md]: they had, where (comma)

File: _posts/2017-12-18-XPDays-Benelux-2017.md [hunk 26]
```diff
@@ -0,0 +1,208 @@
...
+Eventually, the top 8 questions were picked and each question owner was going to be the conversation host for each table.
```
Comment (bart-blommaerts) [inline _posts/2017-12-18-XPDays-Benelux-2017.md]: for a different table

Comment (bart-blommaerts) [reply _posts/2017-12-18-XPDays-Benelux-2017.md]: or for one of the tables

File: _posts/2017-12-18-XPDays-Benelux-2017.md [hunk 27]
```diff
@@ -0,0 +1,208 @@
...
+* What is the quest behind the question? Try to get into the question as deep as possible and figure out what the real reason is behind the question.
+* What is missing? Is anything from the previous conversation still missing? Are there any other additional reasons behind the question?
+* What actions can be taken? Decide on concrete actions that can be taken.
+
+Each question was assigned a time box of 15 minutes and you were only allowed to talk about that specific question during that time box.
+After each question, the other people at the table moved around to another table to help with someone else's issue.
+So, after the exercise, every non-question owner was able to help three different persons by helping with a different question each time.
+This exercise is called a [World Cafe](http://www.artofhosting.org/world-cafe/){:target="_blank"}.
+
+The outcome of the exercise was a group of people happy to have some concrete actions to start working with to try and solve their issues.
```
Comment (bart-blommaerts) [inline _posts/2017-12-18-XPDays-Benelux-2017.md]: people, happy (comma)

File: _posts/2017-12-18-XPDays-Benelux-2017.md (hunk 10 above)
Comment (Deryaa) [reply _posts/2017-12-18-XPDays-Benelux-2017.md]: Haha yup, die opmerking ook al van Yannick gekregen, neem ik zeker mee voor de volgende post :) Thanks!
